osutil/sync_dir_test.go

```diff
@@ -0,0 +1,192 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package osutil_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+
+	. "gopkg.in/check.v1"
+
+	"github.com/ubuntu-core/snappy/osutil"
+)
+
+type EnsureDirStateSuite struct {
+	dir  string
+	glob string
+}
+
+var _ = Suite(&EnsureDirStateSuite{glob: "*.snap"})
```

Comment:
I'd just replace it by "*.foo" instead, to drop the direct implication that it needs to be inline with what the real call site is using.